openedx/core/djangoapps/credit/api.py

```diff
@@ -111,6 +111,37 @@ def get_credit_requirements(course_key, namespace=None):
     ]
 
 
+def get_credit_requirement(course_key, namespace, name):
```

Comment:
I'm conflicted by these API functions. I don't see the value added by thin wrappers around utility methods which already exist on the model. @wedaly, if the model already defines its own interface, is there a need for these functions?